src/main/java/core/basesyntax/shopimpl/service/IOdataFileService.java

```diff
@@ -0,0 +1,75 @@
+package core.basesyntax.shopimpl.service;
+
+import core.basesyntax.model.shopstrategy.ShopTransactionsType;
+import core.basesyntax.shopimpl.entity.DataRecord;
+import core.basesyntax.shopimpl.entity.Fruit;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.util.ArrayList;
+import java.util.List;
+
+public class IOdataFileService {
+    private static final String FILE_DELIMITER = ",";
```

Comment:
What if we want to parse .tsv files or other file formats? Also, some validation on file extensions would be nice